### PR TITLE
Implemented azimuthToBearing

### PR DESCRIPTION
**Pull request by basvdijk**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Have read [How To Contribute](https://github.com/Turfjs/turf/blob/master/CONTRIBUTING.md#how-to-contribute).
- [X] Run `npm test` at the sub modules where changes have occurred.
- [X] Run `npm run lint` to ensure code style at the turf module level.

Submitting a new TurfJS Module.

- [x] Overview description of proposed module.
- [x] Include JSDocs with a basic example.
- [x] Execute `./scripts/generate-readmes` to create `README.md`.
- [x] Add yourself to **contributors** in `package.json` using "Full Name <@GitHub Username>".

### CODE REVIEW

**Review by rowanwins — APPROVED** [review]
👍 @basvdijk

**Comment by JamesLMilner** [comment]
I think we can merge this :)

File: packages/turf-helpers/index.ts [hunk 1]
```diff
@@ -665,6 +665,18 @@ export function bearingToAzimuth(bearing: number): number {
...
+/**
+ * Converts any azimuth angle from the north line direction (positive clockwise)
+ * and returns an angle between -180 and +180 degrees (positive clockwise), 0 being the north line
+ *
+ * @name azimuthToBearing
+ * @param {number} angle between 0 and 360 degrees
+ * @returns {number} bearing between -180 and +180 degrees
+ */
+export function azimuthToBearing(angle: number): number {
+  return angle < 0 ? angle + 360 : angle;
```
Comment (rowanwins) [inline packages/turf-helpers/index.ts:677]: Pretty sure this isn't working as expected @basvdijk now that I've run CI and seen the failing tests.

**Review by rowanwins — CHANGES_REQUESTED** [review]
Requires a tidyup of the function so that the tests pass

File: packages/turf-helpers/index.ts (hunk 1 above)
Comment (stebogit) [reply packages/turf-helpers/index.ts:677]: this works 😃:
```js
function azimuthToBearing(angle) {
  angle = angle % 360;
  if (angle > 0) return angle > 180 ? angle - 360 : angle;
  return angle < -180 ? angle + 360 : angle;
}

test('azimuthToBearing', t => {
  t.equal(azimuthToBearing(0), 0);
  t.equal(azimuthToBearing(360), 0);
  t.equal(azimuthToBearing(180), 180);

  t.equal(azimuthToBearing(40), 40);
  t.equal(azimuthToBearing(40 + 360), 40);

  t.equal(azimuthToBearing(-35), -35);
  t.equal(azimuthToBearing(-35 - 360), -35);

  t.equal(azimuthToBearing(255), -105);
  t.equal(azimuthToBearing(255 + 360), -105);

  t.equal(azimuthToBearing(-200), 160);
  t.equal(azimuthToBearing(-200 - 360), 160);

  t.end();
});
```

**Review by mfedderly — COMMENTED** [review]
I'm open to merging this but can you update this branch and incorporate the suggestions from stebogit? Thanks!

**Comment by basvdijk** [comment]
@mfedderly just added the fixes according to comment of @stebogit

**Comment by mfedderly** [comment]
It looks like you picked up a merge conflict along the way can you knock that out too?

**Comment by basvdijk** [comment]
@mfedderly somehow I can't get it to work. I keep getting issues, merge conflicts and I am currently unable to make it work. I guess it has something to do with being 100 commits behind after a year.

**Comment by mfedderly** [comment]
It might have been that you just needed to pull from the turf repo instead of your fork? If you're just opening a new PR can you @ me so I see it? Thanks.

**Comment by basvdijk** [comment]
@mfedderly Created https://github.com/Turfjs/turf/pull/2674 to replace this PR

**Pull request closed without merging**